TUTORIAL.md

```diff
@@ -271,3 +271,155 @@ First, you need to get the pn-chrono project. It is served on GitHub as well and
     <img width="281" alt="image" src="https://user-images.githubusercontent.com/128436/189910227-3c671b1b-6bbe-4c71-9e8d-77e34ab34017.png">
 
 4. Wait for the next block and check the balance.
+
+
+communicate between multiple nodes
+----------------
+
+### Check before you begin
+Please check the store address of root and peer before starting. PN_StorePath is the address of the store.
+
+
+> root node store path(PN_StorePath) : /tmp/planet-node-chain  
+peer node store path(PN_StorePath) : /tmp/planet-node-chain-a
+
+### 1. Excute root peer node
+```shell
+$ PN_StorePath=/tmp/planet-node-chain dotnet run --project PlanetNode
+```
+### 2. Copy root chain for using same genesis block & Setting peer node chain
+```shell
+$ cp -r /tmp/planet-node-chain /tmp/planet-node-chain-a
+```
+> If you want to know if they want to have the same hash, see the code below.
+```cs
+// /planet-node/Libplanet.Headless/Hosting/SwarmService.cs
```

Comment:
````suggestion

If you would like to verify that two nodes share the same genesis block, you might be able to modify the code so that the nodes output the hash string of the genesis block as below, in planet-node/Libplanet.Headless/Hosting/SwarmService.cs:

```cs
````

Might be better to have non-code outside of the code block.